frontend/src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,51 @@
+import { Link } from 'react-router-dom';
+import styles from './Footer.module.css';
+import logoPlaceHolder from '../../assets/logoPlaceHolder.png';
+
+export const Footer = () => {
+  return (
+    <div className="container">
+      <footer>
+        <div>
+          <img src={logoPlaceHolder} alt="logo" />
+        </div>
+        <div className={styles.footerInfoContainer}>
+          <div>
+            <Link
+              to="/terms-of-service"
+              className={styles.footerText}
+              id={styles.tos}
+            >
+              Terms of Service |
+            </Link>
+            <span>
+              {' '}
```

Comment:
i think the convention is to prefer spacing with css > random whitespace like this

Reply:
gotcha I went ahead and added the css instead